API/Services/MetadataService.cs

```diff
@@ -23,15 +23,15 @@ public MetadataService(IMetadataDatabaseSettings settings)
             _storedMetadata = database.GetCollection<StoredMetadata>(settings.MetadataCollectionName);
         }
 
-        public async Task<StoredMetadata> GetMetadata(string name)
+        public async Task<object> GetMetadata(string name)
         {
             return await Task.FromResult(_storedMetadata.Find(data => data.Name == name).FirstOrDefault());
         }
     }
 
     public class MetadataServiceMocked : IMetadataService
     {
-        public async Task<StoredMetadata> GetMetadata(string name)
+        public async Task<object> GetMetadata(string name)
```

Comment:
Why are these just of the base object-class, not StoredMetadata-objects?

Reply:
Fixed through async